robocop/checkers/lengths.py

```diff
@@ -20,43 +20,43 @@ class LengthChecker(VisitorChecker):
             "too-long-keyword",
             "Keyword is too long (%d/%d)",
             RuleSeverity.WARNING,
-            ('max_len', 'keyword_max_len', int)
+            ('max_len', 'keyword_max_len', int, 'number of lines allowed in a keyword')
```

Comment:
What about writing all configurables like this?
```suggestion
            (
                'max_len',
                'keyword_max_len',
                int,
                'number of lines allowed in a keyword'
            )
```